modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -11,7 +11,7 @@ You can use an AWS-managed policy or a customer-managed policy to set the bounda
 
 [NOTE]
 ====
-This feature is only supported on Red Hat OpenShift Service on AWS (classic architecture) clusters.
+This feature is only supported on Red{nbsp}Hat OpenShift Service on AWS (classic architecture) clusters.
```

Comment:
This module isn't used in any assemblies.

Reply:
Poor guy. He is lonely.